MobileWallet/Common/Theme.swift

```diff
@@ -82,6 +82,8 @@ struct Colors: Loopable {
 
     // Splash
     let splashTitleColor = UIColor(named: "SplashTitleColor")
```

Comment:
I think refactor the `Color` from the name. Just have it as `splashTitle`. It's in the `colors` struct and the user can see the type also. Just check where else that might be in this file and update those also.